datadog_lambda/tags.py

```diff
@@ -63,6 +74,7 @@ def get_enhanced_metrics_tags(lambda_context):
     return parse_lambda_tags_from_arn(lambda_context.invoked_function_arn) + [
         get_cold_start_tag(),
         "memorysize:{}".format(lambda_context.memory_limit_in_mb),
+        "executedversion:{}".format(lambda_context.function_version),
```

Comment:
See my other comment, to be consistent with CloudWatch metrics, can you make `executedversion` only present when the function is invoked by alias?